packages/express/src/accountsExpress.js

```diff
@@ -22,6 +22,27 @@ const accountsExpress = (AccountsServer) => {
   ? res.status(500).jsonp(err.serialize().message)
   : res.status(500).jsonp({ message: err.message }));
 
+  router.use(async (req, res, next) => {
+    const accessToken = get(req.headers, 'accounts-access-token', undefined) || get(req.body, 'accessToken', undefined);
+    if (!isEmpty(accessToken)) {
+      try {
+        const user = await AccountsServer.resumeSession(accessToken);
+        // eslint-disable-next-line no-param-reassign
+        AccountsServer.userId = () => user.id;
```

Comment:
It removes the need to pass the req.user object around. A use case is where you don't want to account for that req parameter constantly, let's say in some package which assumes you have Accounts.